src/db/sqlite/lrsql/sqlite/record.clj

```diff
@@ -16,6 +17,22 @@
 (u/def-hugsql-db-fns "lrsql/sqlite/sql/update.sql")
 (u/def-hugsql-db-fns "lrsql/sqlite/sql/delete.sql")
 
+;; Schema Update Helpers
+
+(defn- update-schema-simple!
```

Comment:
You'd want to add a `#_{:clj-kondo/ignore [:unresolved-symbol]}` here since I'm getting errors in VSCode. (Of course double check that they're actually defined and usable before you do so.)

Reply:
https://github.com/yetanalytics/lrsql/pull/207/commits/4fc65748134088033d169f93230f1c513b26c5ec